fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AlterTableClauseAnalyzer.java

```diff
@@ -1087,12 +1099,10 @@ public Void visitAddPartitionClause(AddPartitionClause clause, ConnectContext co
         return null;
     }
 
-    private void analyzeAddPartition(OlapTable olapTable, List<PartitionDesc> partitionDescs,
-                                     AddPartitionClause addPartitionClause, PartitionInfo partitionInfo)
+    private void analyzeAddPartitionProperties(OlapTable olapTable, List<PartitionDesc> partitionDescs,
```

Comment:
This is the core location of the analyze of add partition. Add partition is not just properties, why do we need to change the name?